composer.json

```diff
@@ -30,5 +34,11 @@
     "psr-4": {
       "Adyen\\Payment\\Tests\\": "tests/"
     }
+  },
+  "scripts": {
+    "test": [
+      "Composer\\Config::disableProcessTimeout",
+      "vendor/bin/phpunit -c Test/phpunit.xml"
```

Comment:
what are we testing here?

Reply:
The test suites are configured in the phpunit.xml file. It includes everything under the Test directory.